package/MDAnalysis/analysis/encore/clustering/Cluster.py

```diff
@@ -0,0 +1,211 @@
+# Cluster.py --- classes to handle results of clustering runs
+# Copyright (C) 2014 Wouter Boomsma, Matteo Tiberti
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Ensemble representation --- :mod:`MDAnalysis.analysis.ensemble.ensemble`
+=====================================================================
+
+The module contains the Cluster and ClusterCollection classes which are
+designed to store results from clustering algorithms.
+
+:Author: Matteo Tiberti, Wouter Boomsma, Tone Bengtsen
+:Year: 2015--2016
+:Copyright: GNU Public License v3
+:Mantainer: Matteo Tiberti <matteo.tiberti@gmail.com>, mtiberti on github
+
+.. versionadded:: 0.14.0
+
+"""
+
+import numpy as np
+
+
+class Cluster:
```

Comment:
Ok, I hadn't looked at QC but I had read your comment on the current bug-hunt... I wish you much success ;-) !